embedded-oracle-xe/src/test/resources/log4j2.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
remove log4j file, use spring boot properties instead, if you want to configure logging for tests, just create ```application.properties``` and paste there ```logging.level.com.playtika.test=info``` and other settings